doc/source/news/14.md

```diff
@@ -1,5 +1,184 @@
 # News - 14 series
 
+<!-- prettier-ignore-start -->
+(release-14-1-0)=
+## Release 14.1.0 - 2024-10-29
+<!-- prettier-ignore-end -->
+
+### Improvements
+
+###### [status] `status`の結果に`llama.cpp`を追加しました。
+
+以下のように、`status`の結果の`features`に新しく`llama.cpp`の有効/無効を表示するようになっています。
+(`status`の表示は長いので説明に必要な部分のみ抜粋しています。)
+
+```
+status
+[
+  [
+    0,
+    0.0,
+    0.0
+  ],
+  {
+    (省略)
+    "features": {
+      (省略)
+      "llama.cpp": true,
+      (省略)
+    },
+    (省略)
+  }
+]
+```
+
+`llama.cpp`は次に紹介する`language_model_vectorize()`に使用しています。
+
+###### 新しい関数 `language_model_vectorize()` を追加しました。(実験的)
+
+まだ実験的な機能なので安定していません。
+また、この機能はCPUとメモリーを多く使います。
+
+この関数は、与えられたテキストの正規化されたエンべディング（ベクトル表現）を返します。
+エンべディングとは、与えられたテキストの意味情報をベクトルとして表現することです。
+意味をベクトルとして表現できることで、字面ではなく意味ベースの検索ができるようになります。
+
+例えば、「牛乳」と「ミルク」は字面は全然違いますが、同じ意味を持つ言葉です。
+従来でも同義語展開を使えば、「牛乳」と「ミルク」のような字面の異なる意味が同じ言葉を検索できますが、同義語展開用のテーブルをメンテナンスしなければなりません。
+
+意味ベースの検索ができるようになると、同義語展開用のテーブルを準備、メンテナンスせずに意味の近い言葉を検索できるようになります。
+
+###### [column_create] 新しい引数 `--generator` を追加しました。
+
+カラムを自動生成できます。
+他のカラムの値からカラムの値を自動で生成できます。
+この機能のメリットは以下の2点です。
+
+1. わざわざユーザーがテキスト生成処理を管理する必要がありません。
+2. 生成元のカラムさえあれば良いので、`load`するデータ量を減らすことができます。
+
+例えば、他のカラムの値を`language_model_vectorize()`でエンべディングして、その値を自動生成したカラムに格納するということもできます。
+これを利用すれば、ユーザーは検索対象のテキストを`load`するだけで、エンべディングされたデータも同時に格納できるようになります。
+
+### Fixes
+
+###### [{doc}`/server/gqtp`] GQPTクライアントに返却する`INVALID_ARGUMENT`の`RC`の値が`-22`ではなく、`65514`になっていた問題を修正しました。
```

Comment:
```suggestion
###### [{doc}`/server/gqtp`] Fixed an issue where the `RC` value of `INVALID_ARGUMENT` returned to the GQTP client was `65514` instead of `-22`.
```